ui-v2/app/styles/components/composite-row/layout.scss

```diff
@@ -0,0 +1,23 @@
+%composite-row a {
+  display: block;
+  padding-left: 12px;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  border-top: 1px solid;
+  border-right: 1px solid;
+  border-left: 1px solid;
+  box-sizing: border-box;
```

Comment:
Kinda neither here nor there but we could do:

```
padding: 12px
padding-top: 0;
border: 1px solid;
border-bottom: 0;
```

Up to you which you prefer.